component/client/chat/ClientChat.tsx

```diff
@@ -34,27 +33,25 @@ class ClientChat extends React.Component<IClientChatProps, IClientChatState> {
   }
 
   public componentWillUnmount(): void {
-    if (this.roomService) this.roomService.close();
+    room.close();
   }
 
   public render(): JSX.Element {
     console.log("[ INFO ] : ClientChat render url and messages", this.props.url, this.state.messages);
     return (
       <div>
-        <Panel header="Chat" bsStyle="primary" footer={this.messenger}>
+        <Panel header={"Chat at " + this.props.url} bsStyle="primary" footer={this.messenger}>
```

Comment:
Consider using the backtick `Chat at ${this.props.url}` which can help understanding the resulting string